ansible/roles/ovn/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+project_name: "ovn"
+
+#######################
+# Docker
+#######################
+
+ovn_db_image_full: "huikang/ovn-scale-test"
+ovn_chassis_image_full: "huikang/ovn-scale-test"
+
```

Comment:
Would you mind creating an account in docker hub, like "ovn-scale-test"? Then I can upload the image to that repository and avoid using this specific name.

I will update the README about how to use customized image.